hack/sync-kubevirtci.sh

```diff
@@ -48,5 +48,5 @@ if [[ "$download_cluster_up" == true ]]; then
     echo ${kubevirtci_git_hash} >${version_file}
     new_sha=$(getClusterUpShasum)
     echo ${new_sha} >${sha_file}
-    echo "KUBEVIRTCI_TAG=${kubevirtci_git_hash}" >>kubevirtci/cluster-up/hack/common.sh
+    echo "KUBEVIRTCI_TAG="'${KUBEVIRTCI_TAG:-'"${kubevirtci_git_hash}}" >>kubevirtci/cluster-up/hack/common.sh
```

Comment:
Should we override `kubevirtci_git_hash` for the rest of the usages as well when it's provided via an env var?